facebook/jest
# jest-editor-support: Don't add "test" prefix to lone "it" statements

## Summary



`jest-editor-support` reports that snapshots don't exist for tests without a describe. This is due to prefixing the descriptions of un-wrapped `it` statements with `test ` when building the name of snapshots to locate.

## Test plan



This is easiest tested in the `vscode-jest` extension;

1. Create a test which looks like the following;
```JavaScript
it('should match snapshot', () => {
  expect('this is an example').toMatchSnapshot();
});
```
2. Run the test using Jest.
3. Note that the snapshot file created looks like this (it does not have a `test` prefix before the snapshot name);
```JavaScript
// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `"this is an example"`;
```
4. The `vscode-jest` extension will report that the snapshot is missing, even though it is not.
5. Swap out `jest-editor-support` for the one in this PR.
6. The extension will no longer report that the snapshot is missing.
